.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature Request
+description: Suggest an idea for ibis
+title: "feat: superb idea"
+labels: ["feature"]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out a feature request!
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your feature request related to a problem?
+      description: If so, please describe the problem.
+      placeholder: "I am frustated by ..."
+    validations:
+      required: false
+  - type: textarea
+    id: request
+    attributes:
+      description: Describe the solution you'd like and any alternatives you may have considered.
+      placeholder: "I'd like ..."
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: Ibis Version
+      description: What version of ibis are you running?
+      placeholder: "Ex: 3.2.0"
+    validations:
+      required: true
+  - type: textarea
+    id: backends
+    attributes:
+      label: Backend(s)
+      description: What backend(s) are you using, if any?
+      placeholder: "Ex: DuckDB"
+    validations:
+      required: false
+  - type: checkboxes
```

Comment:
<img width="940" alt="Screenshot 2022-11-11 at 16 51 06" src="https://user-images.githubusercontent.com/12058428/201376998-b53a3410-7cb4-42dd-a34e-e4e943072ce4.png">
I like to add a question to the forms for a new feature whether the person is interested in contributing. If so, it means that this ticket may need to be prioritized so as not to discourage the person from contributing.

Reply:
Great idea, I'll push up a PR for that :)